src/Avalonia.Themes.Fluent/Accents/BaseDark.xaml

```diff
@@ -139,6 +139,7 @@
     <SolidColorBrush x:Key="SystemControlErrorTextForegroundBrush" Color="{StaticResource SystemErrorTextColor}" />
     <SolidColorBrush x:Key="SystemControlTransientBorderBrush" Color="#000000" Opacity="0.36" />
     <!--<AcrylicBrush x:Key="SystemControlTransientBackgroundBrush" BackgroundSource="HostBackdrop" TintColor="{StaticResource SystemChromeAltHighColor}" TintOpacity="0.8" FallbackColor="{StaticResource SystemChromeMediumLowColor}" />-->
+    <SolidColorBrush x:Key="SystemControlTransientBackgroundBrush" Color="{StaticResource SystemChromeMediumLowColor}" />
```

Comment:
@danwalmsley I added this brush with value from AcrylicBrush.FallbackColor. It matches style on UWP with disabled fluent effects (including acrylic).

Reply:
Good, we are trying to implement acrylic after 0.10